### ISSUE TITLE
ACT-Rule for page title may be at odd with WCAG 2 interpretation of page titles

### ISSUE THREAD

**Issue by mraccess77** [w3c/wcag#980]
The rule https://github.com/act-rules/act-rules.github.io/blob/remove-page-title-example/_rules/html-page-title-2779a5.md seems to allow multiple title elements and use of title outside of head element in support of the HTML spec.  I understand the ACT rules don't have to map to WCAG success criteria -- but they should be harmonized.

Does WCAG 2 allow for multiple titles (it's not accessibility supported) and can you use the title anywhere?  The new rule seems to say yes -- but traditional WCAG wisdom has said no.  It would be good to get consensus on this.

**Comment by Ryladog** [comment]
Seems to me the title element has to be used in the head element...

**Comment by patrickhlauke** [comment]
HTML 5.2 defines what UAs should do if they find more than one `<title>`, and if it's outside of the `<head>`. UAs that follow the spec will, unless I'm mistaken, always take the very first `<title>` they encounter as the one they use/expose as the page title. And provided UAs all handle this the same way, it is accessibility supported (transparently, because the UA will handle it and expose it just the same way that a `<title>` in the `<head>` is).

"Traditional WCAG wisdom" should be updated to reflect the reality of implementations. I note though that there's no failure techniques related to pages having multiple `<title>` elements or those elements not being in the `<head>`.  And neither does the understanding doc for 2.4.2 say anything about this either. So is this more anecdotal wisdom, or am I missing a place where it disallows this explicitly in WCAG?

**Comment by Ryladog** [comment]
For 2.2 updating this would be good.

WCAG 2.0 arrived almost a decade before the HTML 5.2 spec...:-)

**Comment by mraccess77** [comment]
I can't find any understanding documents that forbid it.  The understanding docs are not clear in what title to look for to test but nothing in them specifically flags this.  I can close the issue.
